packages/kotti-ui/source/kotti-field-toggle/KtFieldToggle.vue

```diff
@@ -72,7 +72,8 @@ export default defineComponent({
 				forceUpdateKey: forceUpdateKey.value,
 			})),
 			onInput: (newValue: boolean | undefined) => {
-				field.setValue(newValue ?? null)
+				if (!field.isDisabled && !field.isLoading)
```

Comment:
Shouldn't it be as below? Right now, `forceUpdate()` is always executed.
```suggestion
				if (field.isDisabled || field.isLoading) return
```

Reply:
I wanted to go with Santiago's way of doing but you saw a reason not to ? can you explain why forceUpdate() would be needed in that case? Is there an edge case you had in mind where a forceUpdate would be required?

Reply:
That much I can understand but I'm confused as to how the field is even updated in the first place

Reply:
I keep getting confused by this myself
@santiagoballadares see the JS doc on `useForceUpdate`

Reply:
TBH I'm confused, this: 

> whenever the field updates itself it needs to be forceUpdated

is clear, but I don't follow this:

> to restore the old value

shouldn't it be: `to make the native HTML input display the new value`?

I get that whenever the field's value is updated it needs to be forceUpdated but here when the field is loading all interactions are/should be blocked so that the field's value does not get updated in the first place, so no need to forceUpdated it bc the value didn't change, no?

Anyway, I suppose there's no harm in forceUpdating the field in either case, it'll just re-render.